libs/src/LSPlugin.ts

```diff
@@ -75,20 +73,34 @@ export interface LSPluginPkgConfig {
   mode: 'shadow' | 'iframe'
   themes: Theme[]
   icon: string
-
-  [key: string]: any
+  /**
+   * Alternative entrypoint for development.
+   */
+  devEntry: unknown
+  /**
+   * For legacy themes, do not use.
+   */
+  theme: unknown
```

Comment:
I'm not _actually_ sure what these fields (and `effect`, `iir`, and `lsr`) are for, but uhhh this seems about right